src/View/Cell.php

```diff
@@ -176,6 +176,17 @@ public function render($template = null)
         }
 
         $render = function () use ($template) {
+            try {
+                $reflect = new ReflectionMethod($this, $this->action);
+                $reflect->invokeArgs($this, $this->args);
+            } catch (ReflectionException $e) {
+                throw new BadMethodCallException(sprintf(
+                    'Class %s does not have a "%s" method.',
+                    get_class($this),
+                    $this->action
+                ));
+            }
```

Comment:
This will also allow the action to change the template. I see that as a :+1: though.